src/shared/tpm2-util.c

```diff
@@ -5656,7 +5656,7 @@ int tpm2_define_policy_nv_index(
                         .nvPublic = {
                                 .nvIndex = nv_index,
                                 .nameAlg = TPM2_ALG_SHA256,
-                                .attributes = TPM2_NT_ORDINARY | TPMA_NV_WRITEALL | TPMA_NV_POLICYWRITE | TPMA_NV_OWNERREAD,
+                                .attributes = TPM2_NT_ORDINARY | TPMA_NV_WRITEALL | TPMA_NV_POLICYWRITE | TPMA_NV_OWNERREAD | TPMA_NV_NO_DA,
```

Comment:
the PIN used here might be user supplied, in which case we shouldn't set the flag here

Reply:
Turns out I misunderstood how this part works. I've tested this thoroughly without `TPMA_NV_NO_DA`. Even when the TPM is in DA lockout, it only affects `systemd-pcrlock make-policy` (both creating and updating) -- enrolling and unlocking with a pcrlock policy work even under DA lockout.

I'll undo this line.